test/not_null.t.cpp

```diff
@@ -443,7 +541,24 @@ CASE( "not_null<>: Allows to construct from a non-null underlying pointer (uniqu
     EXPECT( &*p == raw );
 }
 
-CASE( "not_null<>: Allows to move from a not-null pointer (unique_ptr)" )
+CASE( "not_null<>: Returns underlying pointer with get() (unique_ptr)" )
```

Comment:
This test case is not named well: when "transparent get" is on, we're not returning the underlying pointer (I've used the term "underlying pointer" to refer to T previously).

Reply:
Would "Returns underlying pointer or raw pointer with get()" be better?